WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -343,6 +343,24 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
             mPasswordEditText.setText(mPost.getPassword());
         }
 
+        updateStatusSpinner();
```

Comment:
Is this change relevant anymore? Not sure but, it might be left over from previous versions of this PR.

Same for changes in `PostUtils.java` and `WPAlertDialogFragment.java`.